playpen/ansible/roles/dev/files/bashrc

```diff
@@ -82,6 +82,16 @@ pclean() {
     done
     popd
 
+    # Scrub out the Squid cache
+    if [ -d /var/spool/squid ]; then
+        sudo systemctl stop squid
+        sudo rm -rf /var/spool/squid
+        sudo mkdir --context=system_u:object_r:squid_cache_t:s0 --mode=750 /var/spool/squid
+        sudo chown squid:squid /var/spool/squid
+        sudo squid -z
```

Comment:
On Mon, Mar 21, 2016 at 11:56:48AM -0700, Jeremy Cline wrote:
> It definitely doesn't fail on F23 (which as we all know is old news, it's all about F24 these
> days)…

Jeremy, what we all know is that Fedora 24 is old news. It's all about
F25 bro!

-- 
Randy Barlow
irc:   bowlofeggs